tests/panic/external_C/incorrect_SO_file.rs

```diff
@@ -0,0 +1,16 @@
+//@rustc-env: RUST_BACKTRACE=0
+//@only-target-linux
+//@only-on-host
+//@compile-flags: -Zmiri-external_c_so_file=tests/external_C/badpath.so
+//@normalize-stderr-test: "note: rustc.*running on.*" -> ""
+
+extern "C" {
+    fn printer();
+}
+
+fn main() {
+    unsafe {
+        // calling a function from a shared object file that doesn't exist
+        printer();
+    }
```

Comment:
Why should this panic?

`tests/panic` is for tests where, if you run that code in real rustc, it panics. I don't think this is such a test.

This seems to test `-Zmiri` argument parsing. I don't think we have any tests for that, and before we do we would first want to switch the argument parser to a more friendly way of reporting errors.

So please just remove this test for now.